src/Atc.Azure.Options/ServiceOptions.cs

```diff
@@ -7,7 +7,7 @@ public partial class ServiceOptions
 {
     public string ApplicationInsightsConnectionString { get; set; } = string.Empty;
 
-    [Obsolete("ApplicationInsightsConnectionString should be used instead")]
+    [Obsolete("ApplicationInsightsConnectionString should be used instead - see https://docs.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net")]
```

Comment:
Nice, I was just about to comment that it would be nice supply a link to the issue/explanation